storybook/preview.tsx

```diff
@@ -122,6 +122,7 @@ const preview = {
               ["Overview", "*"],
             ],
           ],
+          "Actions",
           "Overlays",
```

Comment:
Just realised why our stories aren't ordering themselves correctly. We can either remove Actions and Overlays (and any future component group) and just rely on the wildcards, or we'll have to add some additional arrays under each group

```suggestion
        order: [
          "Introduction",
          "Guides",
          [
            "App starter",
            "Layout and spacing",
            "Tailwind",
            [
              "Overview",
              "Getting Started",
              "Configuration",
              "Working with Tailwind",
              "*",
              "Utility Class References",
              ["Overview", "*"],
            ],
          ],
          // This will group new folder like actions, overlays, etc in the same order
          "*",
          ["*", ["*", ["Usage Guidelines", "API Specification", "*"]]],
          "Components",
          ["Kaizen Provider", "*"],
          "Pages",
        ],
```

Reply:
Still can't figure out why the stories come first but its a step in the right direction I suppose 🤔

Reply:
Would you be happy for me to leave this for your PR where you add all of the groupings?

Reply:
Yeah that's a good idea, we can make a seperate PR for that to make it clear :)